regparser/layer/terms.py

```diff
@@ -3,7 +3,12 @@
 from itertools import chain
 import re
 
-from inflection import pluralize
+import inflection 
+try:
+    del inflection.PLURALS[inflection.PLURALS.index(('(?i)(p)erson$', '\\1eople'))]
+except ValueError:
+    pass
```

Comment:
It's entirely possible. It seems like all of the instances of "person" that are pluralized in CFPB regs it becomes "persons" (and I can see the semantic difference between "persons" and "people"). It would be fantastic if we could match both, but I don't see an easy way to do that with the inflection library.